packages/server/validators/users.js

```diff
@@ -18,6 +18,23 @@ const userIdValidator = [
     }),
 ];
 
+const userIdBodyValidator = [
+  body("userId")
+    .notEmpty()
+    .withMessage("User ID cannot be empty")
+    .blacklist("<>")
+    .isUUID()
+    .withMessage("Invalid user ID provided")
```

Comment:
Add a `.bail()` check to this UUID check so it looks like this.
```js
  //...
  .isUUID()
  .withMessage("Invalid user ID provided")
  .bail()
  // .custom( ....
```

Adding `.bail()` makes it so the validator terminates immediately after failing any previous checks instead of continuing with validation. This is useful to stop validation early before hitting our database for the custom validator which helps with performance.

![image](https://github.com/cppsea/icebreak/assets/36950420/4d239a9d-43e4-467e-b596-52e8d2a75e74)